src/index.js

```diff
@@ -289,7 +289,7 @@ export function load ({
     }
   } else { // Load only specified states into the local Redux state tree
     states.forEach(function (state) {
-      if (localStorage[namespace + '_' + state] !== 'undefined') {
+      if (typeof localStorage[namespace + '_' + state] !== 'undefined') {
```

Comment:
A humble suggestion for this validation is to use a positive path first
```suggestion
      if (localStorage[namespace + '_' + state] && localStorage[namespace + '_' + state] !== 'undefined') {
```
This validation verify if the key exists and avoids a an convertion JSON parse error if this key receive the string `undefined`

Reply:
Hi @Deivis , I implemented your pull request locally. Good news, it worked! Better news, it kicked me off to investigate further the cause of the undefined value. I was able to determine that the undefined value shouldn't have been occurring in the first place and squashed that bug once and for all (fingers crossed). I'll explain in more depth in this issue https://github.com/kilkelly/redux-localstorage-simple/issues/24
Thank you!